### DISCUSSION TITLE
Syntax Highlighting

### DISCUSSION THREAD

**Discussion by byounghok** [texstudio-org/texstudio#3024] [Q&A]
I have 2 questions on the syntax highlighting of the TXS setting. 

1. I set many formats on differnet color as I want. If I uninstall the TXS program from my computer, then how I can recover this setting? Isn't there any way to recover my own color setting automately? 
2. I cannot find a menu to change the color to indicate the selection of text in the editor. Any idea?

Thanks.

**Comment by sunderme** [comment]
1) texstudio.ini contains all information (https://github.com/texstudio-org/texstudio/wiki/Frequently-Asked-Questions#where-are-the-settings-stored)
2) selected text colors are taken from the stylesheet, i.e. they are the same as a menu selection would be.

**Reply by byounghok** [reply]
Thanks for the answer! 
Here is a follow-up question on #2. What I meant was the highlight text color shown in the backgound of the text, when we double-click the mouse's left botton on the text. I found an example (pls, click the following link) by googling. You can see the screen shot on the top of the page. In this example, the highlight text color is blue, and I would like to change this color to different one. 

https://www.google.com/imgres?imgurl=https%3A%2F%2Fwww.texstudio.org%2Fimages%2Fbookmarks.png&tbnid=IFmYwBk5lkA24M&vet=12ahUKEwjLkLmd8-v9AhWUAt4KHTiXDgcQMygQegUIARDcAQ..i&imgrefurl=https%3A%2F%2Fwww.texstudio.org%2F&docid=-fRiHpD5j4uFmM&w=400&h=225&q=texstudio%20block%20text&ved=2ahUKEwjLkLmd8-v9AhWUAt4KHTiXDgcQMygQegUIARDcAQ

**Comment by sunderme** [comment]
same thing, both colors are taken from the style sheet (txs/options/style) , see also https://texstudio-org.github.io/background.html#style-sheets
